components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -336,6 +339,16 @@ export default class PostList extends React.PureComponent {
             }
         }
 
+        if (scrollUpdateWasRequested) { //if scroll change is programatically requested i.e by calling scrollTo
+            //This is a private method on virtlist
+            const postsRenderedRange = this.listRef.current._getRangeToRender(); //eslint-disable-line no-underscore-dangle
```

Comment:
It would be nice not to call a private method here, but I guess we control the library, so it's less likely that we'll break it by accident

Reply:
I needed to do it on scroll so there wasn't much of choice, Although there is another method which gives out the rendered values which can be stored in state and can be consumed for scenarios as this. I feel it is ok for now though i want to reevaluate if i see any other scenarios needing these values